framework/src/main/java/org/apache/drill/test/framework/Utils.java

```diff
@@ -667,6 +694,34 @@ private static boolean updateDrillStoragePlugin(String filename, String pluginTy
     return false;
   }
 
+  /**
+   * Updates storage plugin for drill
+   *
+   * @param filename
+   *          name of file containing drill storage plugin
+   * @param pluginType
+   *          type of plugin; e.g.: "dfs", "cp"
+   * @param enabled
+   *          value of the "enabled" field in a storage plugin
+   * @return true if operation is successful
+   */
+  private static boolean changeDrillStoragePluginStatus(String filename, String pluginType, String fsMode, boolean enabled) {
```

Comment:
Hey @denysord88 , Is this used only for enabling / disabling storage plugins? 

If so, i'm trying to understand the steps below which replace maprfs and location. I'm not sure why that's needed and I don't think this is generic enough for any storage plugin. If one intends to use hdfs for example, or if the plugin doesn't have the location field. 

Can we have this method exclusively enable / disable plugins?

Reply:
We need this method to disable s3 storage plugin after all tests are done.
Since we are stopping Minio server at the end of the test run, we can't leave S3 storage plugin enabled. It will cause SQL Exception before the next tests will be started.

I didn't want to overload current **_updateDrillStoragePlugin(String filename, String pluginType, String fsMode)_** method because disabling storage plugin will be used only for the s3 storage plugin at the end of the test run if S3 tests aren't excluded.

Reply:
Still not clear why we are doing the other modifications if we are only interested in disabling the plugins. And I think we can then have a simpler disablePlugin method which can be used to disable any plugin.

Reply:
Done. Updated this method to the disableStoragePlugin(String filename, String pluginType) and removed all replacements instead of the "enabled" option.